doc/api/http.md

```diff
@@ -107,6 +107,14 @@ http.get({
 });
 ```
 
+<!-- YAML
+added: v10.14.0
+-->
+
+The maximum size of HTTP/1 headers is 8KB and the maximum time that the HTTP/1
+headers could be received is limited to 40 seconds.
```

Comment:
The 40 seconds timeout is only present on the server, not on the client.

Reply:
@mcollina do you think we should put the text on the top HTTP section (regardless of server or client) but comment on the timeout that its server only, or is it better to put the notices in their own sections (which means we'll repeat the same 8KB headers limit in both server and client)

Reply:
I think we should put the text in each of the respective section.

Reply:
Updated and reserved only the comments about the headers size because I noticed that `server.headersTimeout` already properly specifies the 40 seconds default.